scispacy/umls_utils.py

```diff
@@ -0,0 +1,133 @@
+from typing import List, Dict
```

Comment:
Please add small tests for these functions. You can add test fixtures at `tests/fixtures`.